schemas/usgeo_tables.sql

```diff
@@ -3,7 +3,7 @@ DROP TABLE IF EXISTS meta.us_fips CASCADE;
 CREATE TABLE meta.us_fips (
     "id" SERIAL PRIMARY KEY,
     "fips" BIGINT,
-    "name" VARCHAR(30),
```

Comment:
What is the purpose for this change?

Reply:
After I removed the fips filter (which ignored things like territories), there was at least one name that was longer than 30 characters. It is a relatively small table so I didn't think the `TEXT` type would cause any issues.

Reply:
makes sense -- This is ok with me!